wagtail/sites/templates/wagtailsites/create.html

```diff
@@ -1,7 +0,0 @@
-{% extends "wagtailadmin/generic/create.html" %}
```

Comment:
Let's keep this file with just this line to lower the impact for people updating to the patch version 🙂

Reply:
Done, good call.